scripts/readme_generator.py

```diff
@@ -0,0 +1,56 @@
+import os
+import json
+import dateparser
+import re
+
+
+LEVEL1 = "#"
+LEVEL2 = "##"
+LEVEL3 = "###"
+LEVEL4 = "####"
+
+def markdown_generator(event):
+    file_text = []
+    event_date = event["horario"]
+    event_place = event["local"]
+    event_name = event["evento"]
+    event_title = f"{LEVEL1} {event_name} ({event_date})\n"
+    file_text.append(event_title)
+
+    file_text.append(f"Local: {event_place}\n")
```

Comment:
Acho que o horário é muito grande pra ficar no título. Poderia deixar ele junto com o local, que nem no site. 
```suggestion
    event_title = f"{LEVEL1} {event_name}\n"
    file_text.append(event_title)

    file_text.append(f"Local: {event_place}\n")
    file_text.append(f"Horário: {event_date}\n")
```